catalog/app/containers/Bucket/PackageTree.js

```diff
@@ -318,8 +322,8 @@ function DirDisplay({
             {!noDownload && (
               <FileView.ZipDownloadForm
                 className={classes.button}
-                label="Download package"
-                suffix={`package/${bucket}/${name}/${hash}`}
+                label={path ? 'Download directory' : 'Download package'}
```

Comment:
```suggestion
                label={path ? 'Download sub-package' : 'Download package'}
```

Reply:
I think that the user can ask "what do you mean by sub-package", and the answer will be "it's just the directory you currently browsing". So, I think it's better to explicitly articulate what user will get

Reply:
yeah, but maybe we should differentiate plain s3 directory from package directory, idk. @akarve wdyt?